src/index.html

```diff
@@ -13,20 +13,34 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="styles/style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <input
+        class="search search--big page__search"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="get"
+      data-qa="small"
+    >
+      <input
+        class="search search--small page__search"
```

Comment:
Similar to the previous comment, 'search' and 'search--small' should be meaningful and descriptive of the content or function, not just the style or size.